src/crypto/index.js

```diff
@@ -619,20 +619,35 @@ Crypto.prototype.getEventSenderDeviceInfo = function(event) {
 Crypto.prototype.setRoomEncryption = async function(roomId, config, inhibitDeviceQuery) {
     // if we already have encryption in this room, we should ignore this event
     // (for now at least. maybe we should alert the user somehow?)
-    const existingConfig = this._roomList.getRoomEncryption(roomId);
-    if (existingConfig && JSON.stringify(existingConfig) != JSON.stringify(config)) {
-        console.error("Ignoring m.room.encryption event which requests " +
-                      "a change of config in " + roomId);
+    const existingAlg = this._roomEncryptors[roomId];
+    if (existingAlg) {
```

Comment:
It might be nice to keep the console error in the case that we get an encryption event that's different to what we previously had, since this is a sign something fishy is going on with crypto.

Reply:
hasn't it just moved a few lines down? :)

Reply:
oh, no it hasn't. yeah; let's keep this warning in place.

Reply:
:+1: should be ready for another look